ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -188,5 +188,6 @@ conformance_test(
         "--include=IdentityIT",
         "--all-tests",
         "--load-scale-factor=1",
+        "--target-port=8080",
```

Comment:
I don't understand how this can possibly work. What's running on port 8080?

Reply:
[Magic.](https://github.com/digital-asset/daml/pull/6651/files#diff-521e7919379a572a473d45ef017fc0eeR75-R79) Look at the name of the test. :wink:

Reply:
I'll add a comment to mention that the test is supposed to fail if by any change anything's picked up on port 8080. :smile:

Reply:
Aha, thanks! Sounds good.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6651/commits/b32fba94ce96e9da84f8706ed2da90ebb10f1eb6